core/internal/cltest/cltest.go

```diff
@@ -463,6 +463,7 @@ func NewApplicationWithConfig(t testing.TB, cfg chainlink.GeneralConfig, flagsAn
 		CapabilitiesRegistry:       capabilitiesRegistry,
 		CapabilitiesDispatcher:     dispatcher,
 		CapabilitiesPeerWrapper:    peerWrapper,
+		//TODO beholder client?
```

Comment:
/nit - whats your plan here?

Reply:
Presumably we need to pass this around, but we don't have any usages yet :shrug:

Reply:
> Presumably we need to pass this around

Does it have to be passed around or global getter can be used (like OTel prescribes to do with global providers)
to get the instance or provider of the logger/tracer/meter/emitter?

Reply:
Yes it is probably simpler to use the global getting from the beholder package. I will make that change.